src/toniarts/openkeeper/world/creature/CreatureLoader.java

```diff
@@ -108,9 +108,9 @@ public String getCenterIcon() {
 
     public static void setPosition(Spatial creature, Vector2f position) {
         creature.setLocalTranslation(
-                position.x * MapLoader.TILE_WIDTH - MapLoader.TILE_WIDTH / 2f,
+                MapLoader.TILE_WIDTH * (position.x - 0.5f),

```

Comment:
Isn't this formula basically wrong? If our WIDTH isn't 1, this doesn't work at all?